Containerfile.cert-manager-operator.bundle

```diff
@@ -13,8 +13,7 @@ ARG CERT_MANAGER_OPERATOR_IMAGE=registry.redhat.io/cert-manager/cert-manager-ope
     CERT_MANAGER_WEBHOOK_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:e1339ddb2c6925205a386087fb7472e8358eb910bd6d2e3b294bb8612d50f836 \
     CERT_MANAGER_CA_INJECTOR_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:e1339ddb2c6925205a386087fb7472e8358eb910bd6d2e3b294bb8612d50f836 \
     CERT_MANAGER_CONTROLLER_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:e1339ddb2c6925205a386087fb7472e8358eb910bd6d2e3b294bb8612d50f836 \
-    CERT_MANAGER_ACMESOLVER_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:6f8d7f4ce91305709b2fc04fbce44fba0090b179347f6c48feb70a847f3c2ae5 \
-    KUBE_RBAC_PROXY_IMAGE=registry.redhat.io/openshift4/ose-kube-rbac-proxy-rhel9@sha256:6fbeaf03058bb5d6e9e3311a7afe67666e0770532c978b6b57e77ea2851cb085
+    CERT_MANAGER_ACMESOLVER_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:6f8d7f4ce91305709b2fc04fbce44fba0090b179347f6c48feb70a847f3c2ae5
```

Comment:
https://github.com/openshift/cert-manager-operator-release/pull/191/files#diff-26278c63c84d0a95a66f62b3888601074ba8bc2065a6f77482c3ea41feb7af69R31 needs to be removed.